### ISSUE TITLE
Set prop and evars with Omniture

### ISSUE THREAD

**Issue by joshvickerson** [luisfarzati/angulartics#338]
An issue I ran into when using the Adobe Omniture plugin on a project recently is that it didn't appear possible to configure prop or eVar values using this service. This ultimately led to me using a vanilla JS implementation of Omniture tracking mixed with Angular.

Is it possible to set these values with the service? If so, can someone provide me an example? If not, is this something others would benefit from? Willing to look into developing this but a little direction on how to integrate with Angulartics correctly would be welcome.

**Comment by Faisalnwz01** [comment]
hey did you figure it out? i am trying to impement this too

**Comment by joshvickerson** [comment]
I did not implement setting those with Angulartics. What I ended up doing was setting them using the typical Omniture tracking code and just setting up events with Angulartics.

Ideally, it would be possible to set them using the $analytics service because that would be far more flexible but my project timeline did not give me enough time to try implementing this and given the lack of response here I figured it wasn't a high priority for the project so I haven't spent any of my personal time on it.
